parent/pom.xml

```diff
@@ -34,6 +34,7 @@
 
     <properties>
         <bcprov.version>1.78.1</bcprov.version>
+        <bc.fips.version>1.0.2.5</bc.fips.version>
```

Comment:
There is a [2.1.0](https://mvnrepository.com/artifact/org.bouncycastle/bc-fips/2.1.0) version...

Reply:
I will upgrade to latest bc.fips.version